src/main/java/DeadlineTask.java

```diff
@@ -0,0 +1,2 @@
+package PACKAGE_NAME;public class DeadlineTask {
+}
```

Comment:
You might want to add some spaces between the package and the class.
```suggestion
package PACKAGE_NAME;

public class DeadlineTask {
}
```